lib/functionsShellCommandAction.js

```diff
@@ -38,9 +40,20 @@ module.exports = function(options) {
         useColors: true,
       });
 
+      var projectId = getProjectId(options);
+      var functionsInfo = helper.getFunctionsInfo(emulator.triggers, projectId);
```

Comment:
It's cleaner to construct the functionsInfo inside of localFunction.js than here, then you don't have to change the signature of the LocalFunction constructor.  If you are doing this out here because helper.getFunctionsInfo deals with an array, then you can just construct an array of 1 when inside localFunction.js (i.e.` [ trigger ] `)

Reply:
I constructed functionsInfo in functionsShellCommandAction because I needed the options variable to retrieve the projectId. Is there a way to get the projectId inside localFunction that would not require passing it in?

Reply:
After refreshing my memory on what `helper.getFunctionsInfo` actually does, you don't need to use it at all. The purpose of that function is to handle the case where a user deploys the same function to multiple regions

e.g. 
```
exports.myFunction = functions.regions('us-central1', 'us-east1').database.ref('...").....
```
helper.getFunctionsInfo then has to take the exported triggers and turn this one function into 2 functions - one for each region. 

For emulation, the user would want to only have one function emulated anyways, not one for each region. So you don't need to use this method. You should be able to use `trigger` directly for everything you're doing in localFunction.